README.md

```diff
@@ -47,8 +47,26 @@ $ anvil
 
 ### Deploy
 
+Add `LOCAL_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80` to your `.env` file.
+
+From a new terminal run:
+
+```shell
+avril
+```
+Then, open a new terminal and type:
+
 ```shell
-$ forge script script/Counter.s.sol:CounterScript --rpc-url <your_rpc_url> --private-key <your_private_key>
+forge script scripts/local-deploy.sol --rpc-url 127.0.0.1:8545 --broadcast --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+```
+
+Note that the address, is provided by `avril`. In case you need to provide another one, change `LOCAL_PRIVATE_KEY`.
+
+You can find the contract address by using the following command:
+
+```shell
+cat broadcast/local-deploy.sol/31337/run-latest.json | grep -C 1 "CosmoShips"
+
```

Comment:
make it generic deploy

Reply:
Update the comments. 
Simple, minimal

Reply:
I don't agree with that. Documentation, is a key, and for me seperates the good project from the rest.
Having detailed instructions will help anyone, even those with not a great experience in the project and the stack.